### PR TITLE
Code Exchange Upgrade

### LINKED ISSUE #6: Code Exchange quality checklist
The purpose of this checklist is to assess the state of this repository, and bring it to an acceptable level for long-term support by completing the tasks listed below.

> Please check every part of this checklist for us to process your request. You can do this by marking them with [x] instead of [ ].

### Documentation

- [x] Twilio [LICENSE](https://github.com/twilio-labs/code-exchange/blob/master/LICENSE) exists in repository

- [x] [Code of Conduct](https://github.com/twilio-labs/.github/blob/master/CODE_OF_CONDUCT.md) exists in repository

- [x] [CONTRIBUTING](https://github.com/twilio-labs/.github/blob/master/CONTRIBUTING.md) exists in repository 

- [x] No broken links in README.md

- [x] Build status exists at top of README.md

- [x] Set up steps are complete and accurate

- [x] CodeExchange repository is linked in README.md

### GitHub

- [x] Dependabot set up

- [x] Dependabot automerge set up

- [x] Uses GitHub Actions (required for Twilio sample) or other CI

### Code

- [x] Packages/dependencies are up to date

- [x] Code compiles against latest packages/dependencies

- [x] Matches [CodeExchange Guidelines](https://github.com/twilio-labs/code-exchange/tree/master/guidelines) to the best of ability

### Tests

- [x] Functionality tests have been created

- [x] All tests pass

### App

- [x] Confirmed sample behaves as expected (after dependencies have been updated)

- [x] This file has been copied into a wiki entry for the respective repository or added as a GitHub issue

### General

- [x] I've read and followed the contribution guidelines

- [x] I've read and implemented my template according to the respective coding guidelines for my project

- [x] I agree and adhere to the Code of Conduct

- [x] The template has been authored by me / or my organization and will be kept up-to-date. If the template will not longer be maintained, I'll notify this project.

### PR DESCRIPTION
**Pull request by po5i**
Resolves #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,13 +31,7 @@ Begin by creating a configuration file for your application:
...
 ```
 
-Edit `.env` with the four configuration parameters we gathered from above. Export
-the configuration in this file as system environment variables like so on Unix
-based systems:
-
-```bash
-source .env
-```
+Edit `.env` with the five configuration parameters we gathered from above.
```
Comment (bharatmurali) [inline README.md:34]: we should include a step before this to include setting up a virtual environment

File: README.md [hunk 2]
```diff
@@ -89,7 +85,7 @@ There's just a few more steps to get Twilio's voice infrastructure talking to yo
 3. When ngrok starts up, it will assign a unique URL to your tunnel.
 It might be something like `https://asdf456.ngrok.io`. Take note of this.
 
-4. [Configure your TwiML app](https://www.twilio.com/console/phone-numbers/dev-tools/twiml-apps)'s
+4. [Configure your TwiML app](https://www.twilio.com/console/voice/twiml/apps)'s
```
Comment (bharatmurali) [inline README.md:95]: we update this to reference `pip3` and `python3`

File: README.md [hunk 3]
```diff
@@ -103,6 +99,9 @@ We'd recommend Google Chrome or Mozilla Firefox instead.
 
 ![screenshot of phone app](https://s3.amazonaws.com/com.twilio.prod.twilio-docs/images/TwilioClientQuickstart.original.png)
```
Comment (bharatmurali) [inline README.md:107]: This app doesnt seem to work for me. I constantly get `Twilio.Device Error: Invalid Access Token issuer/subject`

Comment (po5i) [reply README.md:107]: Are you testing locally with the `code-exchange-upgrade` branch? 
Are `TWILIO_ACCOUNT_SID`, `API_KEY`, and `API_SECRET` values correct in your `.env` file?

Comment (bharatmurali) [reply README.md:107]: not sure what changed, but this started working for me again. are we correctly setting the environment variables whenever the app is run? or do we expect the user to manually `source .env` after making changes to the file?

Comment (po5i) [reply README.md:107]: It's not needed to `source .env`. This change was introduced in this PR.

**Review by bharatmurali — APPROVED** [review]

**Pull request merged**